pysrc/pip_resolve.py

```diff
@@ -123,7 +123,7 @@ def create_package_as_root(package, dir_as_root):
             dir_as_root[DEPENDENCIES][package_as_root[NAME]] = package_tree
     return dir_as_root
 
-def satisfies_python_requirement(parsed_operator, py_version_str, sys=sys):
```

Comment:
It's bad practice to pass in python modules as parameters. Better use mock for testing - keeps the shipped code clean and allows you to use proper contexts.